integration-tests/camel/invoked/root/camel-websocket-jsr356/pom.xml

```diff
@@ -10,11 +10,6 @@
 
     <artifactId>quarkus-universe-integration-tests-camel-websocket-jsr356</artifactId>
 
-    <!-- See: https://github.com/apache/camel-quarkus/issues/2262 -->
-    <properties>
-        <skipTests>true</skipTests>
```

Comment:
We probably want to keep this since the tests will fail on Quarkus SNAPSHOT builds.

Reply:
Probably because you're not building with `quarkus.version` set to `999-SNAPSHOT`? The breaking changes are in the master branch. Not the 1.12 release.

Reply:
Oh, I see. Then keeping those tests active is correct as long as we are now building for the release 1.12 of the platform, isn't it?

I have disabled them before I read your message, because it is quite late for any experiments now.

Reply:
No, we need the daily tests with Quarkus master to pass too.